src/components/ResultItem/ResultItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+type ResultItemProps = {
+  className?: string;
+  meta?: string;
+  title: string;
+  href: string;
+  text?: string;
```

Comment:
Can we do these in alphabetical order please? Just to stay consistent (realised I need to update some components I've written to do this, too!)

Only exception would be a `props` prop, which would handle any "other" props (see the Hamburger component for an example)

Reply:
Done.